flatgeobuf/flatgeobuf.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>get the shape, not just the length</p><p>also are we going to add 1 more cell selecting only some columns?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>